static/js/publisher/metrics/graphs/snapInstalls.js

```diff
@@ -0,0 +1,219 @@
+import Chart from "chart.js";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { schemeCategory10 } from "d3-scale-chromatic";
+import { scaleOrdinal } from "d3-scale";
+
+class SnapInstalls extends Component {
```

Comment:
This component does a lot: loads data, renders error/loading state, draws graph...

It would be nice if we could at least split it into two: one "SnapInstallsChart" that would be just a stateless component that would draw a chart based on data passed in props, other one "SnapInstallsController" would deal with data handling and it would render "SnapInstallsChart" once data is fetched.

Does that make sense, or does chart.js handle it better when done in place?

Reply:
It can be split, not sure we want to do it before deciding if we are good to go with chart.js or not?

Reply:
@jpmartinspt Yes, I guess it's not worth putting more work if we are not sure.

Would be good to get UX and design feedback first to see if chart.js is feasible of doing what we need design wise.